projects/kit/components/input-number/input-number.component.ts

```diff
@@ -83,6 +83,7 @@ export class TuiInputNumberComponent
             allowDecimal: decimal !== 'never',
             decimalLimit: precision,
             requireDecimal: decimal === 'always',
+            autoCorrectDecimalSymbol: false,
```

Comment:
I believe this flag is no longer needed and we can refactor `tuiCreateNuberMask` to simplify it. At least please leave a TODO so we change that in 3.0